cumulusci.yml

```diff
@@ -87,7 +87,7 @@ tasks:
     mockaroo_contacts:
         class_path: cumulusci_mockaroo.tasks.MockarooToSql
         options:
-            schema: 'HEDA: Contacts'
```

Comment:
@dashattuck @jstvz  FYI note, I reverted the change at the top of this file so the project is still "HEDAP" and package is "HEDA" until release engineering completes the repository rename.

Reply:
I'm pretty sure this is also dead. Only reference in `git grep` was in April 2017 (909cc26). I'll clean this up as part of the repo rename.